packages/angular_devkit/build_angular/src/builders/web-test-runner/index.ts

```diff
@@ -0,0 +1,199 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import { BuilderContext, BuilderOutput, createBuilder } from '@angular-devkit/architect';
+// eslint-disable-next-line import/no-duplicates
+import type * as WebTestRunner from '@web/test-runner';
```

Comment:
Consider using `typeof import('@web/test-runner')` instead.

Reply:
Curious to understand what the benefit is there? Is it just to remove the duplicate import and lint exception? I don't feel that strongly, just wondering if there's an advantage to this I'm not seeing.

I did realize we can use `WebTestRunner.EventEmitter` and drop the second import, so I consolidated those two imports together and dropped the lint comment.

Reply:
Yes, it's most to avoid the duplicate import and lint exception